assets/sass/components/_errors.scss

```diff
@@ -0,0 +1,3 @@
+.error-sub-heading {
+  padding-top: 40px;
+}
```

Comment:
Think its better to use the predefined sass helper functions rather than hardcoding pixel numbers, but most people tend to just use the [override spacing classes](https://design-system.service.gov.uk/styles/spacing/) and avoid separate css rules entirely

Reply:
updated to use the override spacing classes